developer_manual/examples/cron/SomeTask.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace OCA\MyApp\Cron;
+
+use \OCA\MyApp\AppInfo\Application;
+
+class SomeTask
```

Comment:
make it extend `TimedJob` or `QueuedJob`. I suggest `TimedJob` for the example as it's the most common. (yeah I know we discussed this but I looked at examples in core and it's probably better to still add it)